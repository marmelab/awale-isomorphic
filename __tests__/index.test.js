import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from '../pages/index';

describe('Index with Enzyme', () => {
    it('App shows "Hello world!"', () => {
        const app = shallow(
            <App />
        );
        expect(app.find('p').text()).toEqual('Hello World!');
    });
});

describe('Index  with Snapshot Testing', () => {
    it('App shows "Hello world!"', () => {
        const component = renderer.create(<App />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
