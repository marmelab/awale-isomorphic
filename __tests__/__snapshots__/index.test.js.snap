// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match with snapshot 1`] = `
<div
  className="jsx-462691946"
>
  <nav
    className="jsx-462691946 menu"
  >
    <h1
      className="jsx-462691946 menu__h1"
    >
      Awale
    </h1>
    <ul
      className="jsx-462691946 menu__ul"
    >
      <li
        className="jsx-462691946 menu__li"
        onClick={[Function]}
      >
        <a
          className="jsx-462691946 menu__a"
          href="/game"
          id="newGame"
          onClick={[Function]}
        >
          Solo
        </a>
      </li>
      <li
        className="jsx-462691946 menu__li"
        onClick={[Function]}
      >
        <a
          className="jsx-462691946 menu__a"
          href="/game"
          onClick={[Function]}
        >
          With a friend
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
